etc/kernelspecs/dask_python_yarn_remote/kernel.json

```diff
@@ -0,0 +1,23 @@
+{
+  "language": "python",
+  "display_name": "Dask - Python (YARN Remote Mode)",
+  "metadata": {
+    "process_proxy": {
+      "class_name": "enterprise_gateway.services.processproxies.yarn.YarnClusterProcessProxy"
```

Comment:
Have you confirmed that cells can be interrupted correctly?  To confirm, try running the following cell and attempt to interrupt within 60 seconds.  You should get `KeyboardInterrupt` exception.
```
import time
print('begin')
time.sleep(60)
print('end')
```
Also, is the kernel's shutdown happening gracefully?  You'll probably need to check the EG log for that. If Dask/Yarn simply honors the YARN REST api, then it probably will shutdown cleanly.  Given you're able to use the notebook probably means we're good here.  (That's really cool!)

Reply:
Yes, this works fine as is.